index.html

```diff
@@ -2,7 +2,7 @@
 <head>
   <title>Server Timing</title>
   <meta charset='utf-8'>
-  <script src='//www.w3.org/Tools/respec/respec-w3c-common' async class='remove'></script>
+  <script src='//www.w3.org/Tools/respec/respec-w3c' async class='remove'></script>
```

Comment:
w3c-common profile is deprecated. Although, this wasn't causing the linking error.